src/riak_core_console.erl

```diff
@@ -752,6 +753,27 @@ commit_staged([]) ->
                       "plan and try to commit again~n")
     end.
 
+timezone([]) ->
+    TZ = riak_core_metadata:get({riak_core, timezone}, string,
+                                [{default, "not configured"}]),
+    io:format("~s~n", [TZ]);
+timezone([TZString]) ->
+    TZ = jam:process(jam_iso8601:parse_tz(TZString)),
+    attempt_set_timezone(jam:is_valid(TZ), TZ, TZString).
```

Comment:
Because `jam` is designed around pipeline processing, it's very convenient to return the result directly instead of via tuples. The few cases where I do return tuples (see the README) I'd like to find a cleaner way to handle.

I'm not saying no, but neither am I saying yes.